python/tk_multi_publish2/base_hooks/collector.py

```diff
@@ -307,6 +534,43 @@ def process_current_session(settings, parent_item):
         """
         raise NotImplementedError
 
+    def create_properties_widget(self, parent, items):
+        """
+        Creates a Qt widget, for the supplied parent widget (a container widget
+        that is used to display Item Details).
+
+        :param parent: The parent to use for the widget being created.
+        :param items: List of items to create the Settings tab for.
+        :return: A QtGui.QWidget or subclass that displays information about
+            the item and/or editable widgets for modifying the item's
+            properties.
+        """
+
+        if not len(self.plugin.settings["Properties To Display"].value):
+            no_properties_to_display = QtGui.QLabel(parent)
+            no_properties_to_display.setAlignment(QtCore.Qt.AlignCenter)
+            no_properties_to_display.setObjectName("no_properties_to_display")
+            no_properties_to_display.setText("No Properties to display for the selected items.")
+            widget = no_properties_to_display
+        else:
+            widget = CollectorPlugin.PropertiesWidgetController(parent, self, items)
+
+        return widget
+
+    def on_properties_changed(self, settings, items):
+        """
+        Method that runs when the property_changed signal is fired from the property widget.
+
+        :param settings: Settings for the plugin.
+        :param item: Item to run property change hook for.
+        """
+        # raise NotImplementedError
+        # TODO: remove this.
+        print "running on_properties_changed"
```

Comment:
as per comment, remove this :)

Reply:
Will remove it when cleaning up for the merge commits... kept it here for testing.